### PR TITLE
feat: add Sepolia support 

### PR DESCRIPTION
**Pull request by tamtamchik**
Adds minimal Sepolia testnet support, which is required to create a simple vote. Required for #196 and #205.

--- 
> [!WARNING]
> Not all parameters are set in `configs/config_sepolia.py`, and acceptance/regression/snapshot tests are not operational.

### CODE REVIEW

File: utils/voting.py [hunk 1]
```diff
@@ -193,12 +193,13 @@ def confirm_vote_script(
...
-            specific_net=CHAIN_NETWORK_NAME,
-            repeat_is_error=True,
-        )
+        # human_readable_script = decode_evm_script(
+        #     encoded_call_script,
+        #     verbose=False,
+        #     specific_net=CHAIN_NETWORK_NAME,
+        #     repeat_is_error=True,
+        # )
+        human_readable_script = []
```
Comment (TheDZhon) [inline utils/voting.py:202]: ehh, needs to be fixed in the 3rd-party lib

**Comment by tamtamchik** [comment]
```
tests/regression/test_easy_track_factories.py::test_node_operators_activations tests/regression/test_easy_track_factories.py::test_transfer_node_operator_manager =========================== short test summary info ============================
FAILED tests/regression/test_easy_track_factories.py::test_node_operators_activations
FAILED tests/regression/test_easy_track_factories.py::test_transfer_node_operator_manager
===== 2 failed, 280 passed, 14 skipped, 20 warnings in 2068.07s (0:34:28) ======
```
😢

File: utils/voting.py (hunk 1 above)
Comment (tamtamchik) [reply utils/voting.py:202]: Fixed in https://github.com/lidofinance/AVotesParser/releases/tag/v0.5.6

File: README.md [hunk 2]
```diff
@@ -83,26 +83,36 @@ By default, you should start composing new scripts and test using forked network
...
+> You can't run tests on the live networks.
 
 In a typical weekly omnibus workflow, you need only `mainnet-fork` and
 `mainnet` networks. In case of large test campaign on Lido upgrades,
-it also could be useful to go with `goerli` and `goerli-fork` testnets first.
+it also could be useful to go with `holesky` and `holesky-fork` testnets first.
 
+> [!IMPORTANT]
 > **Holesky specifics.**
 > Due to Holesky not being supported by Infura yet, setting RPC URL for Holesky is different. Instead of setting `WEB3_INFURA_PROJECT_ID` env variable set `HOLESKY_RPC_URL`.
```
Comment (TheDZhon) [inline README.md:104]: https://docs.infura.io/api/network-endpoints#ethereum

At the moment of writing, Infura supports both Holešky and Sepolia :eyes:

File: README.md [hunk 3]
```diff
@@ -83,26 +83,36 @@ By default, you should start composing new scripts and test using forked network
...
+>
+> **Sepolia specifics.**
+> Due to Sepolia not being supported by Infura yet, setting RPC URL for Holesky is different. Instead of setting `WEB3_INFURA_PROJECT_ID` env variable set `SEPOLIA_RPC_URL`.
```
Comment (TheDZhon) [inline README.md:107]: see above

**Review by TheDZhon — CHANGES_REQUESTED** [review]
LGTM except for only a few details about Infura RPC

File: README.md (hunk 2 above)
Comment (tamtamchik) [reply README.md]: Great! Updated network config. Re-mapped to Infura RPC.

File: README.md (hunk 3 above)
Comment (tamtamchik) [reply README.md]: Fixed in https://github.com/lidofinance/scripts/pull/208/commits/a5eb329c7dac82d23fceeb2a5d7927864f1e4426

**Review by TheDZhon — APPROVED** [review]
:ship: it

**Pull request merged**